rep-2000.rst

```diff
@@ -709,6 +709,121 @@ Package manager use for dependencies:
+| Middleware Library       | Middleware Provider | Support Level | Platforms                   | Architectures                  |
++==========================+=====================+===============+=============================+================================+
+| rmw_cyclonedds_cpp       | Eclipse Cyclone DDS | Tier 1        | All Platforms               | All Architectures              |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+| rmw_fastrtps_cpp         | eProsima Fast-DDS   | Tier 1        | All Platforms               | All Architectures              |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+| rmw_connext_cpp          | RTI Connext         | Tier 1        | All Platforms except Debian | All Architectures except arm64 |
+|                          |                     |               | and OpenEmbedded            |                                |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-DDS   | Tier 2        | All Platforms               | All Architectures              |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minimum language requirements:
+
+- C++17
+- Python 3.7
+
+
+Dependency Requirements:
+
++-------------+---------------------------------+------------------------------------------------------+
+|             |        Required Support         |                 Recommended Support                  |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Package     |  Ubuntu Focal  |   Windows 10** |     MacOS**   | Debian Buster  | OpenEmbedded**      |
++=============+================+================+===============+================+=====================+
+| CMake       |     3.16.3     |     3.14.4     |     3.14.4    |      3.13.4    | 3.16.1 / 3.12.2**** |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| EmPY        |                                      3.3.2                                             |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Gazebo      |     11.0.0*    |      N/A       |     11.0.0    |      11.0.0*   |        N/A          |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Ignition    |    Citadel*    |      N/A       |            Citadel*            |        N/A          |
```

Comment:
Should Galactic be paired with Ignition Dome?

Pinging @chapulina who might know.

Reply:
Thanks for the ping, we discussed this back in November and I forgot to mention it here.

It will be Edifice, to be released in March 2021 and maintained until March 2022 (we could extend support by 2 months to May 2022 to match Galactic if there's enough interest).

```suggestion
| Ignition    |    Edifice*    |      N/A       |            Edifice*            |        N/A          |
```

We're also working on Windows support, I'll update it here if we get to it on time.